### ISSUE TITLE
Wrong format on selecting colors

### ISSUE THREAD

**Issue by alexkleinubing** [claviska/jquery-minicolors#180]
Hello, see the pen:
http://codepen.io/anon/pen/JYxQvK

Whats do it?

**Comment by claviska** [comment]
Whoops. The `format` and `keywords` settings were never intended to have `data-` shortcuts like `opacity` has. In fact, I'd like to eventually phase that syntax out completely to make the API more consistent. I should have caught that. (This is a syntax Bootstrap introduced; it's convenient but can also be confusing.)

This was an oversight due to the way controls are instantiated in the demo. The demo uses the `data-format` shortcut to populate the settings object. So instead of the settings object being set, the `data-` shortcuts were being used instead. Coincidentally in the demo, everything appeared to work properly while testing—but only because the `data-` attributes matched the desired settings.

In short, what I've done in b5b3947 fixes this issue and removes support for `data-` shortcuts for `format` and `keywords`. Normally this would be a breaking change, but since this behavior was never intended I consider it a bug.

@JoomliC — since you were a major contributor to this feature I want to CC you so you can be aware of this fix so nothing breaks on your next update. As always, feedback and testing is welcome.

**Comment by alexkleinubing** [comment]
I understand.
Taking advantage, another question would be why the format be changed during and after choosing colors: http://codepen.io/anon/pen/JYxQvK

**Comment by claviska** [comment]
That should be fixed as well. Can you try again with the master? Can't remember if I updated the demo or not.

**Comment by JoomliC** [comment]
@claviska Hi!
And thanks to mention me :+1: 
As i've not yet had time to create the pull request for Joomla, i will check all this soon when i will take some time to do this (before end of this year... ;-) )
